qa-niaid.planx-pla.net/etlMapping.yaml

```diff
@@ -33,6 +33,7 @@ mappings:
       - name: required_idp
       - name: external_url
       - name: created_datetime
```

Comment:
we should probably remove `created_datetime` since we're replacing the order

Reply:
Makes sense - I'll do that in the follow-up PR since I still need to sort by it until I add the new fields on the data.